model/task/task.go

```diff
@@ -1699,9 +1705,15 @@ func ActivateDeactivatedDependencies(tasks []string, caller string) error {
 		return errors.Wrap(err, "updating activation for dependencies")
 	}
 
+	logs := []event.EventLogEntry{}
 	for _, t := range tasksToActivate {
-		event.LogTaskActivated(t.Id, t.Execution, caller)
+		logs = append(logs, event.GetTaskActivatedEvent(t.Id, t.Execution, caller))
 	}
+	grip.Error(message.WrapError(event.LogManyEvents(logs), message.Fields{
+		"message":  "problem logging task activated events",
```

Comment:
nit: should this one and the one on 1791 be just "logging task..."?

Reply:
(and 1842)

Reply:
I think it's actually okay like this. The style guide says that the `"problem"` rule would only apply to wrapped error messages (i.e. strings used for `errors.Wrap` or `errors.Wrapf`) since they tend to wrap several error messages up the call stack. In this case, this just logs as a single error message, so it's not an issue.